From 99da10fa1c771bd692bf00b6c4ccf2a5c1818bdb Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Tue, 08 Jan 2013 06:35:42 +0000
Subject: background: Remove obsolete plugin

The background drawing has now moved to gnome-shell, allowing
smoother integration, animated backgrounds, etc.

https://bugzilla.gnome.org/show_bug.cgi?id=686549

Keep the schemas around since these are used by Unity-settings-daemon.
---
Index: gnome-settings-daemon-3.7.90/data/gnome-settings-daemon.convert
===================================================================
--- gnome-settings-daemon-3.7.90.orig/data/gnome-settings-daemon.convert	2013-02-23 13:52:19.012459246 -0500
+++ gnome-settings-daemon-3.7.90/data/gnome-settings-daemon.convert	2013-02-23 13:52:19.000459246 -0500
@@ -14,6 +14,10 @@
 active = /apps/gnome_settings_daemon/plugins/a11y-keyboard/active
 priority = /apps/gnome_settings_daemon/plugins/a11y-keyboard/priority
 
+[org.gnome.settings-daemon.plugins.background]
+active = /apps/gnome_settings_daemon/plugins/background/active
+priority = /apps/gnome_settings_daemon/plugins/background/priority
+
 [org.gnome.settings-daemon.plugins.clipboard]
 active = /apps/gnome_settings_daemon/plugins/clipboard/active
 priority = /apps/gnome_settings_daemon/plugins/clipboard/priority
Index: gnome-settings-daemon-3.7.90/data/org.gnome.settings-daemon.plugins.gschema.xml.in.in
===================================================================
--- gnome-settings-daemon-3.7.90.orig/data/org.gnome.settings-daemon.plugins.gschema.xml.in.in	2013-02-23 13:52:19.012459246 -0500
+++ gnome-settings-daemon-3.7.90/data/org.gnome.settings-daemon.plugins.gschema.xml.in.in	2013-02-23 13:52:19.000459246 -0500
@@ -11,6 +11,7 @@
     </key>
     <child name="a11y-keyboard" schema="org.gnome.settings-daemon.plugins.a11y-keyboard"/>
     <child name="a11y-settings" schema="org.gnome.settings-daemon.plugins.a11y-settings"/>
+    <child name="background" schema="org.gnome.settings-daemon.plugins.background"/>
     <child name="clipboard" schema="org.gnome.settings-daemon.plugins.clipboard"/>
     <child name="color" schema="org.gnome.settings-daemon.plugins.color"/>
     <child name="cursor" schema="org.gnome.settings-daemon.plugins.cursor"/>
@@ -54,6 +55,18 @@
       <_summary>Priority to use for this plugin</_summary>
       <_description>Priority to use for this plugin in gnome-settings-daemon startup queue</_description>
     </key>
+  </schema>
+  <schema gettext-domain="@GETTEXT_PACKAGE@" id="org.gnome.settings-daemon.plugins.background" path="/org/gnome/settings-daemon/plugins/background/">
+    <key name="active" type="b">
+      <default>true</default>
+      <_summary>Activation of this plugin</_summary>
+      <_description>Whether this plugin would be activated by gnome-settings-daemon or not</_description>
+    </key>
+    <key name="priority" type="i">
+      <default>97</default>
+      <_summary>Priority to use for this plugin</_summary>
+      <_description>Priority to use for this plugin in gnome-settings-daemon startup queue</_description>
+    </key>
   </schema>
   <schema gettext-domain="@GETTEXT_PACKAGE@" id="org.gnome.settings-daemon.plugins.clipboard" path="/org/gnome/settings-daemon/plugins/clipboard/">
     <key name="active" type="b">
